operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,203 @@
+# Zuul users guide
+
+## Prerequisites
+
+1. Repository is known by your zuul instance
+2. Basic ansible knowledge
+3. Basic yaml knowledge
+4. zuul-client installed
+
+## Who is it for?
+
+You may have heard about zuul and may ask yourself if it is capable to support you.
+Basically everything you use ansible for can be done using zuul. That is not always
+a good thing since you may get careless and your workload will exceed the CI/CD concept.
+
+If you find yourself doing things under the following list you are at the right place.
+
+1. Code testing
+2. Deployment tests using IaC
+
+If you want to, let's say, monitor something using zuul, that is possible but not the
+intended use case.
+
+## Where do I start?
+
+Right in your projects repository! The only prerequisite you need to fulfill, is that
+your repository you want zuul to work on is known by zuul. This is done by the zuuls
+tenant configuration. To update this configuration you need access to the zuul instance
+or ask an administrator for help.
```

Comment:
We should add exactly what is to be done, eg.

"Contact the SCS Zuul operators at foo@" or establish a similar workflow as with the OpenStack projects flow for the gx-scs demonstrator.

Reply:
I would keep it as it is

```
Right in your projects repository! The only prerequisite you need to fulfill, is that
your repository you want zuul to work on is known by zuul. This is done by the zuuls
tenant configuration. To update this configuration you need access to the zuul instance
or ask an administrator for help.
```

and add 

```
To do so get in touch by contacting .......
```

Would that be sufficient? And if so which e-mail address / phone number / matrix account to add? :grin:

Reply:
We concluded in today's team OPS call that we will document as follows:

Please open an issue to have your repository added to zuul using the proper template in the issues repository.
PR for the template is here: https://github.com/SovereignCloudStack/issues/pull/436